dipy/reconst/ivim.py

```diff
@@ -215,6 +215,9 @@ def __init__(self, gtab, split_b_D=400.0, split_b_S0=200., bounds=None,
             e_s += "The IVIM model requires signal measured at 0 bvalue"
             raise ValueError(e_s)
 
+        if gtab.b0_threshold > 0:
+            warnings.warn('The default b0_threshold > 0 and is now set to 50')
```

Comment:
I don't understand this warning. 

As I user, I would infer that this means that you have changed my b0_threshold to 50. 

I would propose to raise an Error (instead of the warning) with the following string:

"The IVIM model requires a measurement at b==0. As of version 0.15, the default `b0_threshold` for the `GradientTable` object is set to 50, so if you used the default settings to initialize the `gtab` input to the IVIM model, you may have provided a `gtab` with `b0_threshold` larger than 0. Please initialize the `gtab` input with `b0_threshold=0`"

Reply:
@arokem Thank you so much for this! I shall refactor accordingly!